plan/logical_plan_builder.go

```diff
@@ -225,6 +225,9 @@ func (b *planBuilder) buildJoin(join *ast.Join) LogicalPlan {
 	b.optFlag = b.optFlag | flagPredicatePushDown
 	leftPlan := b.buildResultSetNode(join.Left)
 	rightPlan := b.buildResultSetNode(join.Right)
+	if b.err != nil {
```

Comment:
Should we log this error? When will we meet error here?

Reply:
Considering this sql "SELECT * FROM events e JOIN (SELECT clock FROM events e2 GROUP BY e2.source) e3 ON e3.clock=e.clock" , the join.Right will meet error when set sql_mode ONLY_FULL_GROUP_BY

Reply:
so we should check `b.err` after `leftPlan := b.buildResultSetNode(join.Left)` as well?

Reply:
I don't find the case that join.Left cause b.err != nil